### ISSUE TITLE
Support creating and placing blocks

### ISSUE THREAD

**Issue by acbramley** [jhedstrom/drupalextension#313]
It would be great to have step definitions similar to nodes, taxonomy terms, etc for blocks!

You could also use the config step code to set the region/weight/visibility as well

**Comment by jhedstrom** [comment]
Good idea! If somebody finds themselves working on this, I think the way to go would be to add a new `BlockContext` class.

**Comment by subhojit777** [comment]
I guess we have to handle the block placing from the DrupalDriver itself. BlockContext will just use the method to place the block.
